milestone2.ts

```diff
@@ -92,23 +98,33 @@ let findAverage: GetNumber; // Define lambda function here
   Return -1 if no one is from San Francisco.
   Hint: Use a lambda function as a value
 */
-let findIndexAns: GetNumber; // Code here
+let findIndexAns: GetNumber = data => {
+  const index = data.findIndex(entry => entry.city === "SF")  // finds the index of someone from SF in array
+  return index
+}; // Code here
 
 //console.log(findIndexAns(userData));
 
 // Question 3: Filtering data
 /* Use data handling function(s) to find all of the volunteers from California (CA) over an age threshold n
  */
 type GetVolunteers = (data: Volunteer[], minAge: number) => Volunteer[];
-let findCAOverN: GetVolunteers; // Code here
+let findCAOverN: GetVolunteers = (data, minAge) => {
+  const city_filter = data.filter(x => x.state === "CA")  // creates array of students from CA
+  const age_filter = city_filter.filter(x => x.age > minAge) // further filters by age
```

Comment:
You can combine these filters with a logical and (&&)
```suggestion
  const city_filter = data.filter(x => x.state === "CA" && x.age > minAge)  // creates array of students from CA
```